include/rocksdb/utilities/backupable_db.h

```diff
@@ -116,6 +119,12 @@ struct BackupableDBOptions {
   // Default: INT_MAX
   int max_valid_backups_to_open;
 
+  // If true, backup SST filenames consist of file_number, crc32c, db_session_id
+  // if false, backup SST filenames consist of file_number, crc32c, file_size
+  //
+  // Default: false
+  bool new_naming_for_backup_files;
```

Comment:
I think the default should be the new scheme, as there's no forward/backward compatibility issue within RocksDB. There's a chance of people opting out for a few reasons (external dependency on naming scheme; longer names bad; slight performance regression due to reading SST file property), so I think we should have the option available.

Maybe "share_files_with_checksum_unsafe_legacy_naming". A long name might discourage people from using it. ;)

Reply:
Let me set the default to using the new scheme :)